totes

```diff
@@ -9,32 +9,49 @@ get_render_command() {
   fi
 }
 
+get_notes_path() {
+  # location based on XDG directory specification
+  # ref - https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+  local notes_path="${XDG_DATA_HOME:-$HOME/.local/share}/totes"
+  if [[ ! -d $notes_path ]]; then
+    echo "The notes directory - $notes_path - does not exist."
+    # shellcheck disable=2016
+    echo 'Try running the `install.sh` script again'
+    # checks if this script is in a repo (should be true if it is symlinked)
+    if git rev-parse HEAD > /dev/null 2>&1; then
+      # display full path to install script if we can find it
+      echo "path: $(git rev-parse --show-toplevel)/install.sh"
+    fi
+    exit 1
+  fi
+  printf "%s" "$notes_path"
+}
+
+get_suppressed_notes_registry_spec() {
+  printf "%s/suppressed_note_registry" "$(get_notes_path)"
+}
+
 # Determine the next note to render
 get_note_to_render() {
   note_to_render="$1"
   if [[ -z $1 ]]; then
-    # location based on XDG directory specification
-    # ref - https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
-    notes_path="${XDG_DATA_HOME:-$HOME/.local/share}/totes"
-
-    if [[ ! -d $notes_path ]]; then
-      echo "The notes directory - $notes_path - does not exist."
-      # shellcheck disable=2016
-      echo 'Try running the `install.sh` script again'
-      # checks if this script is in a repo (should be true if it is symlinked)
-      if git rev-parse HEAD > /dev/null 2>&1; then
-        # display full path to install script if we can find it
-        echo "path: $(git rev-parse --show-toplevel)/install.sh"
-      fi
-      exit 1
+    suppressed_notes_registry_spec="$(get_suppressed_notes_registry_spec)"
+    if ! [[ -f "$suppressed_notes_registry_spec" ]]; then
+      # the suppressed_note_registry must not be empty
+      echo "dummy-text-required-here" > "$suppressed_notes_registry_spec"
     fi
 
-    # ref - https://github.com/koalaman/shellcheck/issues/1247
-    # shellcheck disable=2012
-    note_to_render="$(ls -At -- "$notes_path"/*.md | tail -n 1)"
+    notes_path="$(get_notes_path)"
+    # shellcheck disable=2010
+    note_to_render="$(ls -At -- "$notes_path"/*.md | grep -vFf "$suppressed_notes_registry_spec" | tail -n 1)"
     # update last read time to cycle through notes
   fi
-  echo "$note_to_render"
+  printf "%s" "$note_to_render"
+}
+
+suppress_note() {
+  # Consider only writing file if it's not already in suppressed_notes
+  echo "$1" >> "$(get_suppressed_notes_registry_spec)"
```

Comment:
Agree we should check this (could be in a follow-up PR if you prefer for any reason), should be as simple as:

```
grep -sq "$1" "$(get_suppressed_notes_registry_spec)" || {
  echo "$1" >> "$(get_suppressed_notes_registry_spec)"
}
```